examples/quickstart-huggingface/requirements.txt

```diff
@@ -1,6 +1,5 @@
-datasets~=2.13.1
-evaluate~=0.4.0
-flwr~=1.4.0
-numpy~=1.21.1
-torch~=2.0.1
-transformers~=4.30.2
+flwr>=1.0, <2.0
+torch==1.13.1
+transformers==4.30.0
+evaluate==0.4.0
+scikit-learn==1.0
```

Comment:
```suggestion
scikit-learn==1.3.1
```

Reply:
w/ `1.0` installation fails.

Reply:
I used python 3.10.9.

Reply:
I didn't try the `requirements.txt` install

Reply:
I updated the requirements.txt as well and run pip install and it works in both 3.8 and 3.10.